src/modules/Dashboard/features/Respondents/Popups/EditRespondentPopup/EditRespondentPopup.tsx

```diff
@@ -96,29 +97,59 @@ export const EditRespondentPopup = ({
       <>
         {isLoading && <Spinner uiType={SpinnerUiType.Secondary} noBackground />}
         <StyledModalWrapper>
-          <form onSubmit={handleSubmit(submitForm)} noValidate>
-            <StyledController>
-              <InputController
-                fullWidth
-                name="nickname"
-                control={control}
-                label={t('nickname')}
-                data-testid={`${dataTestid}-nickname`}
-              />
-            </StyledController>
-            <StyledController>
-              <InputController
-                fullWidth
-                name="secretUserId"
-                control={control}
-                label={t('secretUserId')}
-                onChange={handleChangeSecretId}
-                error={!!hasServerError}
-                data-testid={`${dataTestid}-secret-user-id`}
-              />
-            </StyledController>
-          </form>
-          {hasServerError && <StyledErrorText>{getErrorMessage(error)}</StyledErrorText>}
+          <Box
+            component="form"
+            sx={{ display: 'flex', flexDirection: 'column', gap: 2.4 }}
+            onSubmit={handleSubmit(submitForm)}
+            noValidate
+          >
```

Comment:
Nit 😇 
```suggestion
          <StyledFlexColumn
            component="form"
            sx={{ gap: 2.4 }}
            onSubmit={handleSubmit(submitForm)}
            noValidate
          >
```

Reply:
I'd prefer to leave this as-is. The wrapping of a StyledComponent around a MUI component breaks the polymorphic type validation, which is being used to validate the types of HTML attributes passed to the rendered `<form>` element here (ie, the `noValidate` attr). 

While I haven't dug into it too much, my hunch is that the error is related to conflicting functionality in MUI and Emotion, which MUI is using for styling. Emotion supports an `as` prop (which still works with MUI components, despite type errors), whereas MUI supports the same feature via a `component` prop.

Applying the changes you suggest would also require manually re-asserting `StyledFlexColumn` as a different component type, which would require an intermediary variable, like...

```ts
const StyledFlexColumnForm = StyledFlexColumn as StyledComponent<BoxProps<'form'>>;
```

Reply:
Wow, good investigation there Justin. This choice to use `component` vs `as` sure makes things needlessly awkward indeed, so yeah, totally understandable not to use the utility. Honestly, I'm not even a fan of the pattern we have in place here for bundling style utilities. I'd be more in favour of something simpler that facilitates composition, which also isn't possible in the current structure. Something like this, for example:

```tsx
const bodyLarge: SxProps = {
  fontSize: variables.font.size.lg,
  lineHeight: variables.font.lineHeight.lg,
  fontWeight: variables.font.weight.regular,
};

const ellipsis: SxProps = {
  textOverflow: 'ellipsis',
  overflow: 'hidden',
  whiteSpace: 'nowrap',
};

<Box sx={{ ...bodyLarge, ...ellipsis }}>
  // …
</Box>
```

That would be big refactor, so not up for discussion obviously. 😅

Reply:
I totally agree with you. 

I haven't checked this very thoroughly, but I'm a bit worried about creating deeply-nested JSX trees as we create components with `styled` to wrap other components created with `styled` and so-on. Objects passed to `sx` would have to be parsed on mounting, but at least by comparison it's just the single instance of that occurring.

And yeah, so long as we're doing CSS in JS, I _much_ prefer the advantages object notation provides regarding type-checking, intellisense, and out-of-the-box syntax highlighting that string literals don't.